tests/python/frontend/tflite/test_forward.py

```diff
@@ -1043,74 +1049,118 @@ def _test_convolution(
 
 def test_forward_convolution():
     for quantized in [False, True]:
-        _test_convolution(
-            [4, 8, 8, 176], [1, 1, 176, 32], [1, 1], [1, 1], "SAME", "NHWC", quantized=quantized
-        )
-        _test_convolution(
-            [4, 17, 17, 19], [3, 3, 19, 19], [1, 1], [2, 2], "VALID", "NHWC", quantized=quantized
-        )
-        _test_convolution(
-            [4, 17, 17, 124], [1, 1, 124, 19], [1, 1], [1, 1], "SAME", "NHWC", quantized=quantized
-        )
-        _test_convolution(
-            [4, 17, 17, 12], [3, 3, 12, 32], [1, 1], [2, 2], "VALID", "NHWC", quantized=quantized
-        )
+        for fp16_quantized in [False, True]:
+            if quantized and fp16_quantized:
+                continue
```

Comment:
Why are we skipping the fp16_quantized test

Reply:
fp16 quantized path doesn't go through quantized path. 
quantized=True and fp16_quantzied=True goes through same code as quantized=False and fp16_quantize=True. So I just wanted to reduce the time test time here.

Reply:
Should I remove the skipping code?

Reply:
Sorry for the delay. Yes, please. Can you restructure the test such that there is no need of skipping code. One might be confused while reading the code otherwise.

Reply:
Sorry for the delay. Yes, please. Can you restructure the test such that there is no need of skipping code. One might be confused while reading the code otherwise.

Reply:
I have addressed your request.